components/mpas-seaice/src/shared/mpas_seaice_column.F

```diff
@@ -10141,6 +10149,35 @@ subroutine check_column_package_configs(domain)
             messageType=MPAS_LOG_CRIT)
     endif
 
+    ! check config_snow_redistribution_scheme
+      if ((trim(config_snow_redistribution_scheme) == "ITDrdg" .or. &
+         trim(config_snow_redistribution_scheme) == "ITDsd") .and. .not. config_use_effective_snow_density) then
```

Comment:
should these configuration flag checks also be in mpas_seaice_icepack.F?

Reply:
Yes.  I'll add those.